backend/settings.py

```diff
@@ -215,12 +215,48 @@
 }
 
 TINYMCE_DEFAULT_CONFIG = {
-        'menubar': True,
-        "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
-        "toolbar": "formatselect | bold italic backcolor | "
-            "bullist numlist outdent indent | removeformat | code | help",
-        "default_link_target": "_blank",
-    }
+    'menubar': True,
+    "plugins": "advlist,autolink,code,lists,link,anchor,insertdatetime,media,table,paste,wordcount",
+    "toolbar": "bold italic backcolor | "
+        "bullist numlist outdent indent | removeformat | code | help",
+    "default_link_target": "_blank",
+    "invalid_elements": "span, h1, h2, h3, h4, h5, h6",
```

Comment:
I believe we discussed the inclusion of `span` here before; was there a reason why this needs to be invalid? It seems like the header tags are the main thing we're trying to avoid here.

Reply:
Like shown in the example [here](https://www.w3schools.com/tags/tag_span.asp), the `span` tag is often used to specify the `style` of a particular part of texts (indentation, color, etc.). Excluding `span` from pasted texts was to remove the customized text styles that we may not want to have in our cards.

Reply:
I'm still a bit uncertain about this. As you say, `span` is often used to specify styles, but it's not exactly the root of the problem; if we wanted to, I think we could disallow more styles using `invalid_styles` as has already been done. I guess my preference is to do that: allow `span` and disallow any specific styles we don't want. @jonespm, what do you think?